### ISSUE TITLE
Cannot create Skeleton SvelteKit project

### ISSUE THREAD

**Issue by 1qh** [oven-sh/bun#6446]
[UPDATED]

### What version of Bun is running?

1.0.7+b0393fba6200d8573f3433fb0af258a0e33ac157

### What platform is your computer?

Linux 6.5.7

### What steps can reproduce the bug?

In a environment that does not have `node` installed, following [Skeleton Docs](https://www.skeleton.dev/docs/get-started), I've tried

```sh
bun create skeleton-app my-svelte-app
```
but it exits at second prompt

This is not a problem of [SvelteKit](https://kit.svelte.dev/docs/creating-a-project) at all because
```sh
bun create svelte my-svelte-app
```
also works fine without `node`.

Eventually I have to install `node` to setup the project directory, then remove `node` to make sure it only used `bun` runtime to `bun i` and `bun dev`. 

There's no issue using only `bun` in development, the only problem is when I need to initialize the project with this frameworks it somehow requires `node`

**Comment by Electroid** [comment]
Thanks for reporting, what error do you run into?

**Comment by 1qh** [comment]
@Electroid 

The command exits at the second prompt, I have no idea why.

Basically I cannot create a Skeleton project using bun, while pnpm works normally.

**Comment by Masch90** [comment]
@1qh which version of node were you using when running into this issue?

I just ran into it aswell. Turne out I had no node installed. (I am not sure at all why that matters just what I experienced)

**Comment by 1qh** [comment]
@Masch90 It's true, with node installed the command works fine. I experienced this in a no-node environment. But I wonder why

```sh
bun c svelte app
```
works without node, while

```sh
bun c skeleton-app app
```
doesn't

**Comment by 1qh** [comment]
The issue has been resolved by Bun 1.0.18. Thank you a lot!
